definitions.yaml

```
@@ -206,6 +209,9 @@ definitions:
             type: integer
             description: |
               The maximum number of cluster nodes as configured
+      region:
+        type: string
+        description: Region where this cluster is running at.
```

```suggestion
        description: Name of the region where this cluster is running at.
```
A bit more aligned with AZ and others